module/apmgopgv10/go.mod

```
@@ -0,0 +1,14 @@
+module go.elastic.co/apm/module/apmgopg
```

```suggestion
module go.elastic.co/apm/module/apmgopgv10
```

Sorry, missed this one before! This should fix the CI failures.

---

Ah, my bad I missed it too.

---

Didn't seam to help